### ISSUE TITLE
Seeker is not working

### ISSUE THREAD

**Issue by edwinharly** [codealchemist/youtube-audio-server#10]

**Comment by codealchemist** [comment]
Able to reproduce on Chrome 70.0.3538.67.
Works OK on Firefox 61.0.2.
All on OSX.

Will investigate what's going on on Chrome.
Thanks for reporting.

**Comment by codealchemist** [comment]
Seeking doesn't work on Chrome because we're just piping the mp3 stream into the express response.
Chrome needs a partial response with byte ranges to properly do seeking.
Wondering why this is not the case for Firefox...

I've found an express middleware that implements partial responses:
https://www.npmjs.com/package/send-seekable

But... I wasn't able to find a performant solution to properly implement it since we need to know the total amount of bytes that will be transferred before sending the response.

I've played converting the stream to a buffer but it is too costly and takes too long to respond.
I accept suggestion to properly do this in a performant way.

Thx!
